python/ray/autoscaler/_private/commands.py

```diff
@@ -470,7 +473,13 @@ def kill_node(config_file: str, yes: bool, hard: bool,
     if not nodes:
         cli_logger.print("No worker nodes detected.")
         return None
-    node = random.choice(nodes)
+    if not _node:
+        node = random.choice(nodes)
+    else:
+        # Kill specific node if supplied via hidden arg.
+        # Used exclusively by autoscaler to stop Ray on nodes of on-prem
+        # clusters.
+        node = _node
```

Comment:
nit: can you flip this to be 
```
if _node:
  node = _node
```

Reply:
yeah good idea -- `if uncommon condition: do uncommon thing`